interactions.go

```diff
@@ -37,16 +37,20 @@ type ApplicationCommand struct {
 	Type              ApplicationCommandType `json:"type,omitempty"`
 	Name              string                 `json:"name"`
 	NameLocalizations *map[Locale]string     `json:"name_localizations,omitempty"`
+	
 	// NOTE: DefaultPermission will be soon deprecated. Use DefaultMemberPermissions and DMPermission instead.
 	DefaultPermission        *bool  `json:"default_permission,omitempty"`
 	DefaultMemberPermissions *int64 `json:"default_member_permissions,string,omitempty"`
 	DMPermission             *bool  `json:"dm_permission,omitempty"`
 
 	// NOTE: Chat commands only. Otherwise it mustn't be set.
-
 	Description              string                      `json:"description,omitempty"`
 	DescriptionLocalizations *map[Locale]string          `json:"description_localizations,omitempty"`
 	Options                  []*ApplicationCommandOption `json:"options"`
+
+	// NOTE: Sometimes return by Discord API. Mustn't be set.
+	NameLocalized 		 *string     `json:"name_localized,omitempty"`
+	DescriptionLocalized *string     `json:"description_localized,omitempty"`
```

Comment:
I think would be better not to include these yet, to avoid confusion. As this implementation does not have the functionality for the language customization (`Accept-Language` and `X-Discord-Locale` headers). 

Plus we have some plans on the implementation of header-based features. So I think we should leave them out, at least before the proper implementation of headers.

Reply:
sounds good